### PR TITLE
Bump to released Pi4J

### PR DESCRIPTION
**Pull request by FDelporte**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,8 +26,8 @@
 
         <!-- Project Dependencies -->
         <java.version>11</java.version>
-        <pi4j.version>2.0-SNAPSHOT</pi4j.version>
-        <slf4j.version>2.0.0-alpha1</slf4j.version>
+        <pi4j.version>2.0</pi4j.version>
+        <slf4j.version>1.7.32</slf4j.version>
         <picocli.version>4.6.1</picocli.version>
         <junit.version>5.7.1</junit.version>
```
Comment (DieterHolz) [inline pom.xml:32]: <junit.version>5.7.2</junit.version>

File: pom.xml [hunk 2]
```diff
@@ -26,8 +26,8 @@
 
         <!-- Project Dependencies -->
         <java.version>11</java.version>
-        <pi4j.version>2.0-SNAPSHOT</pi4j.version>
-        <slf4j.version>2.0.0-alpha1</slf4j.version>
```
Comment (DieterHolz) [inline pom.xml:30]: use the most recent one?
<slf4j.version>2.0.0-alpha5</slf4j.version>

or the same as Pi4J?
<slf4j.version>1.7.30</slf4j.version>

**Review by DieterHolz — COMMENTED** [review]
using the most recent versions?

File: pom.xml (hunk 2 above)
Comment (FDelporte) [reply pom.xml:30]: yep we are using a released version of slf4j, best practice is to not have child dependencies which are alpha, snapshot... in a release

File: pom.xml (hunk 1 above)
Comment (FDelporte) [reply pom.xml:32]: ah didn't see the change in junit yet, any ideas about changes in the new version?

**Comment by DieterHolz** [comment]
Then I would use the same as Pi4J

File: pom.xml (hunk 1 above)
Comment (DieterHolz) [reply pom.xml:32]: only  a bug fix release, I think

**Review by DieterHolz — APPROVED** [review]

**Pull request merged**